react/src/pages/MapProject/MapProject.tsx

```diff
@@ -22,6 +24,41 @@ interface Props {
  */
 const MapProject: React.FC<Props> = ({ isPublic = false }) => {
   const { projectUUID } = useParams();
+  const [initialLoad, setInitialLoad] = useState(true);
+  const [selectedAssetTypes, setSelectedAssetTypes] = useState<string[]>([]);
+  const [startDate, setStartDate] = useState(new Date());
+  const [endDate, setEndDate] = useState(
+    new Date(Date.now() + 24 * 60 * 60 * 1000)
+  );
+
+  useEffect(() => {
+    if (initialLoad) {
+      setSelectedAssetTypes([
+        'Image',
+        'Video',
+        'PointCloud',
+        'Streetview',
+        'Questionnaire',
+        'NoAssetVector',
+      ]);
+      setInitialLoad(false);
+    }
+  }, [initialLoad]);
```

Comment:
I think having an empty dependency array here would ensure that this is only called once but another possible solution would just be to initialize selectedAssetTypes with these variables.

const [selectedAssetTypes, setSelectedAssetTypes] = useState<string[]>(['Image','Video', 'PointCloud', 'Streetview','Questionnaire',  'NoAssetVector',]);

or maybe the following for DRY reasons.

  const [selectedAssetTypes, setSelectedAssetTypes] = useState<string[]>(
    Object.keys(assetTypeOptions)
  );